### PR TITLE
added policies

### PR DESCRIPTION
**Pull request by ch-stark**
Signed-off-by: Christian Stark <cstark@cstark.muc.csb>

### CODE REVIEW

File: community/CM-Configuration-Management/policy-ansible-awx-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                  name: awx-resource-operator
+                  namespace: policies
+                spec:
+                  channel: release-0.1
+                  installPlanApproval: Automatic
+                  name: awx-resource-operator
+                  source: redhat-operators
+                  sourceNamespace: openshift-marketplace
+                  startingCSV: awx-resource-operator.v0.1.1
+---
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-ansible-awx-operator.yaml:58]: A similar policy didn't work for me... make sure this operator installs ok.  I think it needs an OperatorGroup.

File: community/CM-Configuration-Management/policy-cluster-network-sample.yaml [hunk 2]
```diff
@@ -58,3 +58,21 @@ subjects:
 - name: policy-config-network
   kind: Policy
   apiGroup: policy.open-cluster-management.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-policy-config-network
+  namespace: policies
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-cluster-network-sample.yaml]: The namespace shouldn't be hard coded here.  It needs to be removed.

File: community/CM-Configuration-Management/policy-custom-catalog.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-custom-catalog.yaml:3]: The `name` is missing.

File: community/CM-Configuration-Management/policy-kyverno.yaml [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+subjects:
+  - name: policy-install-kyverno
+    kind: Policy
+    apiGroup: policy.open-cluster-management.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-policy-install-kyverno
+  namespace: kyverno
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-kyverno.yaml]: The namespace shouldn't be in the placement rule.

**Review by gparvin — COMMENTED** [review]
Added a few comments.  Thanks for working on this!

File: community/CM-Configuration-Management/policy-cluster-network-sample.yaml (hunk 2 above)
Comment (ch-stark) [reply community/CM-Configuration-Management/policy-cluster-network-sample.yaml]: thanks, good catch

File: community/CM-Configuration-Management/policy-custom-catalog.yaml (hunk 3 above)
Comment (ch-stark) [reply community/CM-Configuration-Management/policy-custom-catalog.yaml:3]: thanks

File: community/CM-Configuration-Management/policy-kyverno.yaml (hunk 4 above)
Comment (ch-stark) [reply community/CM-Configuration-Management/policy-kyverno.yaml]: can we discuss if I can leave it that way?

Comment (gparvin) [reply community/CM-Configuration-Management/policy-kyverno.yaml]: The Policy and it's associated PlacementRule and PlacementBinding all need to be in the same namespace.  We want that namespace to be selected by the customer.
Note that you do have another PlacementRule embedded in the policy in the kyverno namespace.  That one is ok to be in the kyverno namespace, but this one needs to match the root namespace selected by the customer for the policy.

**Review by gparvin — COMMENTED** [review]
Double check the kyverno policy and see if the comment I added makes sense.

**Comment by ch-stark** [comment]
@gparvin thanks I removed the namespace from placement-binding/rule

**Review by gparvin — APPROVED** [review]
Thank you for all these great policies!

**Pull request merged**